src/cov/Coverage.java

```diff
@@ -519,8 +519,13 @@ public static void initCoverage(Class<?> cls) {
    */
   public static Member getMemberContaining(CoverageAtom cov) {
     if (cov == null) throw new IllegalArgumentException("cov cannot be null.");
-    Class<?> cls = Reflection.classForName(cov.getClassName());
-    initCoverage(cls);
+
+    try {
+      initCoverage(Class.forName(cov.getClassName()));
+    } catch (ClassNotFoundException e) {
+      //ignor
```

Comment:
I was maintaining behavior in this case.

Looking at the code, the exception would occur if the class name of the CoverageAtom is not a class name, which appears unlikely. I changed the catch clause to return a null. This would match the intention in the comment.